src/common.rs

```diff
@@ -89,6 +95,10 @@ impl LineEntry {
             None => None,
         }
     }
+
+    pub fn trimmed_string(&self) -> &str {
+        self.raw_string.trim()
```

Comment:
Could you write a test for that function?

Reply:
Done